Microsoft.DotNet.Interactive.Rendering/Formatter.cs

```diff
@@ -358,8 +358,7 @@ private static void TryRegisterDefault(string typeName, Action<object, TextWrite
 
         public static string MimeTypeFor(Type type)
         {
-            _mimeTypesByType.TryGetValue(type, out var mimeType);
-            return mimeType;
+           return  _mimeTypesByType.TryGetValue(type, out var mimeType) ? mimeType : "text/plain";
```

Comment:
Maybe extract a static property `Default` so we don't keep repeating this string?